src/app/project/ontology/ontology.component.ts

```diff
@@ -375,7 +362,7 @@ export class OntologyComponent implements OnInit {
     openResourceClassForm(mode: 'createResourceClass' | 'editResourceClass', resClassInfo: DefaultClass): void {
 
         // set cache for ontology and lists
-        this.setCache();
+        // this.setCache();
```

Comment:
Can this be removed? Same with the other similar commented out line below

Reply:
done in 68380d1